core/drivers/atmel_rstc.c

```diff
@@ -43,6 +45,11 @@ static TEE_Result atmel_rstc_probe(const void *fdt, int node,
 {
 	size_t size = 0;
 
+	if (_fdt_get_status(fdt, node) != DT_STATUS_OK_SEC)
+		return TEE_ERROR_BAD_PARAMETERS;
+
+	matrix_configure_periph_secure(AT91C_ID_SYS);
```

Comment:
is it not an issue if it fails? shouldn't we at least pop an error on the console?

Reply:
@ldts: If it fails, then the `atmel_rstc_available()` will return false. This is used by the code that can invoke reset support. So it is actually "supported", the same can happen if the rstc node is lacking.